scripts/development/build-contenta_jsonapi.sh

```diff
@@ -72,16 +72,19 @@ echo -e "\t********************************"
 echo -e "\n\n\n"
 
 echo "---------------------------------"
-echo " (3/4) One time admin login link "
+echo " (3/3) One time admin login link "
 echo "---------------------------------"
 echo -e "${FG_C}${BG_C} EXECUTING ${NO_C} $DRUSH user-login --no-browser --uri=\"http://127.0.0.1:8888\""
 echo -e "${FG_C}${WBG_C} INFO ${NO_C} Use this link to login as an administrator in your new site:"
 $DRUSH user-login --no-browser --uri="http://127.0.0.1:8888"
 
-echo "-------------------------------------"
-echo " (4/4) Initializing local PHP server "
-echo "-------------------------------------"
-echo -e "${FG_C}${WBG_C} INFO ${NO_C} Server started. Use Ctrl+C to stop it."
-echo -e "${FG_C}${WBG_C} INFO ${NO_C} If you want to start the server manually use:\n\t$DRUSH runserver"
-# Start the server in the background to be able to get the one time login link.
-$DRUSH runserver
+while getopts ":s" opt; do
+  case $opt in
+    s)
+
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
```

Comment:
Let's either:
  1. Ignore the fact that an unknown option was passed.
  1. Exit with a non-zero value to denote error.

I don't have any opinions in which way to go.